www/assets/css/account.css

```diff
@@ -218,7 +232,7 @@ body.theme-b {
 .modal-open {
   visibility: visible;
   display: block;
-  z-index: 1001;
+  z-index: 1000001;
```

Comment:
Look like this is not correct, this value should be greater than **wpt-header** z-index too, but it fails when we have different media width:
- default z-index: https://github.com/catchpoint/WebPageTest/blob/cd6502f48cac364f3b280fbdd7b84579a1538ab7/www/assets/css/wpt-header.css#L38
- z-index with `@media (min-width: 86em)`: https://github.com/catchpoint/WebPageTest/blob/cd6502f48cac364f3b280fbdd7b84579a1538ab7/www/assets/css/wpt-header.css#L419